ui-components/src/locales/general_OLD.json

```diff
@@ -1,870 +0,0 @@
-{
```

Comment:
Do we need to delete this file? It's not getting loaded but I'm hesitant to do it in our fork instead of in the bloom repo.

Reply:
Done.